In [ ]:
import nltk
nltk.download('punkt')
nltk.download('wordnet')
nltk.download('averaged_perceptron_tagger')
nltk.download('vader_lexicon') 
from nltk.sentiment import SentimentIntensityAnalyzer 
sia = SentimentIntensityAnalyzer()
text = "I love this movie, it's so great!"
sentiment = sia.polarity_scores(text) 
if sentiment['compound'] > 0: 
    print("Positive sentiment detected.")
elif sentiment['compound'] == 0: 
    print("Neutral sentiment detected.")
else: 
    print("Negative sentiment detected.")

In [7]:
print(sentiment)

{'neg': 0.0, 'neu': 0.291, 'pos': 0.709, 'compound': 0.8941}


输出结果:{'neg': 0.0, 'neu': 0.291, 'pos': 0.709, 'compound': 0.8941}
分别代表负面情感，中性程度，积极程度，综合评分